### ISSUE TITLE
Print message if there's newer version of filprofiler available on PyPI

### ISSUE THREAD

**Issue by itamarst** [pythonspeed/filprofiler#300]
Via https://github.com/mgrachev/update-informer maybe? Would need to add PyPI support.

**Comment by mgrachev** [comment]
Good news - [`update-informer`](https://github.com/mgrachev/update-informer/releases/tag/v0.3.0) has just been released with PyPi support 🎉
Now it's ready to be integrated with your project 🚀
Thanks a lot for your help 🙏

**Comment by itamarst** [comment]
Mostly just need to get the time to do it. Won't say no to a PR :)

**Comment by mgrachev** [comment]
I've looked at the code 🤯

I'd like to help you, but I don't know where to start 🤔
Also, I don't know Python 🐍

**Comment by itamarst** [comment]
Yeah, it's... not the best project layout, it was my first Rust project :) I'll get to it eventually.
